day_2_core_cnf_clusters/observability/telco-observability.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="telco-observability"]
+= Observability in {product-title}
+include::_attributes/common-attributes.adoc[]
+:context: telco-observability
+:imagesdir: images
+
+toc::[]
+
+{product-title} generates a large amount of data, such as performance metrics and logs from both the platform and the workloads running on it. 
+As an administrator, you can use various tools to collect and analyze all the data available.
+What follows is an outline of best practices for system engineers, architects, and administrators configuring the observability stack.
```

Comment:
Elaborate on `What follows`. IMO, this sentence probably is not required. Do we have something called best practices available in this doc?
Also, avoid listing roles and convert the the sentence as for second person.